### DISCUSSION TITLE
Pros and Cons: Knative Eventing vs Keda

### DISCUSSION THREAD

**Discussion by kmr0877** [kedacore/keda#1216] [Q&A / Need Help]
I am looking for an event-driven auto scaler based on messages in rabbitmq queues and I see  some common points in Keda and Knative Eventing. I am looking for help to understand the Pros and Cons of having one of them from technical point of view, future road map as i see some collaboration between the two.
Cheers

**Comment by tomkerkhove** [comment]
I'm not an expert on that topic but we might want to doc this maybe @zroubalik? EIther under FAQ or a dedicated section. Thoughts?

**Reply by kmr0877** [reply]
@tomkerkhove Agree. I think its worth documenting in dedicated section as most of them may not realize the fundamental differences between them and will be helpful on choosing the right one
